internal/server/iigo.go

```diff
@@ -26,7 +27,7 @@ func (s *SOMASServer) runIIGOEndOfTurn() error {
 		newGameState := s.gameState.GetClientGameStateCopy(clientID)
 		newGameState.ClientInfo.Resources -= tax
```

Comment:
Similarly, the server is not being updated with the new amount of resources that the client has.

Reply:
Agreed, and fixed